webrtc.html

```diff
@@ -9773,6 +9775,8 @@ <h2>Methods</h2>
                 <a>throw</a> an <code>InvalidStateError</code>.</li>
                 <li>Let <var>dtmf</var> be the <code><a>RTCDTMFSender</a></code>
                 associated with <var>sender</var>.</li>
+                <li><a>update the [[\CanInsertDTMF]]</a> internal slot of
+                <var>dtmf</var>.</li>
```

Comment:
[[CanInsertDTMF]] seems redundant with this approach.

Reply:
@aboba As long as the algorithm is synchronous and relies solely on main-thread state, can't we run it whenever the `canInsertDTMF` getter is called, and only then?